include/ppx/grfx/grfx_command.h

```diff
@@ -177,6 +177,21 @@ struct RenderPassBeginInfo
     grfx::DepthStencilClearValue DSVClearValue                          = {1.0f, 0xFF};
 };
 
+// RenderingInfo is used to start dynamic render passes.
+struct RenderingInfo
+{
+    grfx::BeginRenderingFlags flags;
+    grfx::Rect                renderArea = {};
+
+    uint32_t                renderTargetCount                          = 0;
+    grfx::RenderTargetView* pRenderTargetViews[PPX_MAX_RENDER_TARGETS] = {};
+    grfx::DepthStencilView* pDepthStencilView                          = nullptr;
+
+    uint32_t                     RTVClearCount                          = 0;
```

Comment:
Can `renderTargetCount` be used to avoid needing this?

Reply:
yes, changed